client/my-sites/hosting/main.js

```diff
@@ -280,20 +360,41 @@ const Hosting = ( props ) => {
 				{ isJetpack && <QueryJetpackModules siteId={ siteId } /> }
 				<WrapperComponent>
 					<Layout className="hosting__layout">
-						<Column type="main" className="hosting__main-layout-col">
-							<MainCards
+						{ isEnabled( 'layout/dotcom-nav-redesign-v2' ) ? (
+							<AllCards
 								hasStagingSitesFeature={ hasStagingSitesFeature }
 								isAdvancedHostingDisabled={ ! hasSftpFeature || ! isSiteAtomic }
 								isBasicHostingDisabled={ ! hasAtomicFeature || ! isSiteAtomic }
-								isWpcomStagingSite={ isWpcomStagingSite }
 								isBusinessTrial={ isBusinessTrial && ! hasTransfer }
+								isWpcomStagingSite={ isWpcomStagingSite }
 								siteId={ siteId }
 								siteSlug={ siteSlug }
 							/>
-						</Column>
-						<Column type="sidebar">
-							<SidebarCards isBasicHostingDisabled={ ! hasAtomicFeature || ! isSiteAtomic } />
-						</Column>
+						) : (
+							<>
+								<Column className="hosting__main-layout-col">
+									<MainCards
+										hasStagingSitesFeature={ hasStagingSitesFeature }
+										isAdvancedHostingDisabled={ ! hasSftpFeature || ! isSiteAtomic }
+										isBasicHostingDisabled={ ! hasAtomicFeature || ! isSiteAtomic }
+										isBusinessTrial={ isBusinessTrial && ! hasTransfer }
+										isWpcomStagingSite={ isWpcomStagingSite }
+										siteId={ siteId }
+										siteSlug={ siteSlug }
+									/>
+								</Column>
+								<Column>
+									<SidebarCards
+										hasStagingSitesFeature={ hasStagingSitesFeature }
+										isAdvancedHostingDisabled={ ! hasSftpFeature || ! isSiteAtomic }
+										isBasicHostingDisabled={ ! hasAtomicFeature || ! isSiteAtomic }
```

Comment:
I believe this is the only prop needed for `<SidebarCards />`

Reply:
Nice! Forgot to remove it in the cleanup.